chatmaild/src/chatmaild/ini/override-testrun.ini

```diff
@@ -14,3 +14,7 @@ privacy_pdo =
 privacy_supervisor =
     State Commissioner for Data Protection and Freedom of Information of
     Baden-Württemberg in 70173 Stuttgart, Germany.
+privacy_intro =
+    This is the default onboarding server for Delta Chat.
+    If you don't choose to login to an existing email account,
+    you can get one on this server.
```

Comment:
i'd amend/modify this as i have thought about it quite a bit.  It's a bit weird to do this in the ini-file though. 
Maybe more in a call?  There are some related considerations that interfere with this PR a little.

Reply:
actually wait -- this is going to be shown for all *.testrun.org services, right?

Reply:
Yes. of course it only makes sense for nine.testrun.org, but c* and staging are only experimental anyway, so what.

Reply:
call sounds good